client-app/shared/account/components/account-navigation-link-components/link-lists.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <AccountNavigationLink :item="item">
+    <template v-if="isListDetails">
+      <div
+        v-for="list in lists"
```

Comment:
links within a link is weird, this needs to be changed

Reply:
renamed to AccountNavigationItem for now